### ISSUE TITLE
Rethinkdb suddenly status exited

### ISSUE THREAD

**Issue by kirvk548** [rethinkdb/rethinkdb#6701]
Hello Guys,

I'm just newbie to rethinkdb so i have to ask question about this issue. 

We have a cluster of rethinkdb server with 5 nodes on it. All of a sudden, one or two of servers will have a status exited on it, only by restarting the service will make it returned to normal. RethinkDB servers are running on a container with Ubuntu 16.04.3 LTS (Xenial Xerus) installed on it, 4GB RAM, 512 swap and I've already set vm.swappiness=10 in my /etc/sysctl.conf

The question is what is the cause of the issue or any idea on it? 

Attached screenshot is the error that i saw also with no idea what is happening on it

![error-rethinkdb](https://user-images.githubusercontent.com/3414427/44452260-aa3fa300-a628-11e8-8766-21928385a672.PNG)

Thanks for your consideration guys!!

**Comment by srh** [comment]
You found a bug in the clustering logic.

**Comment by kirvk548** [comment]
are they releasing anytime soon fixes on this following issue?

**Comment by srh** [comment]
@kirvk548 No.

**Comment by Extarys** [comment]
@kirvk548 RethinkDB is now known as RebirthDB and once they release the next version (2.4.0), you should upgrade and create an issue for this on the new repo if you still have problems.

:smile:

**Comment by kirvk548** [comment]
Yeah, that's what I'm planning already after releasing their next version.

Thanks :D :D :D

**Comment by gabor-boros** [comment]
Hello @srh ,
Shouldn't we assign this issue to 2.4.x blockers?

**Comment by srh** [comment]
No, the bug is in 2.3.6.

**Comment by gabor-boros** [comment]
Is it fixed in 2.3.7? If yes I would close this issue since there are 2.3.7 packages out on the download server though not for every distro, yet.

**Comment by srh** [comment]
No, it's not fixed.
